Old Pages/Power_Machines.md

```diff
@@ -1,6 +1,6 @@
 
 # Power Machines
- On this page, you will learn how to create and store power, and how you can use it. Wires will be mentioned, but clearly explained here.
+ On this page, you will learn how to create and store power, and how you can use it. Wires will be mentioned, but clearly explained [here].
```

Comment:
split lines and that's a broken link, do it like this: [here](/wiki/old_pages/wires)